examples/notebooks/rag/README.md

```diff
@@ -0,0 +1,91 @@
+# RAG with Data Prep Kit
+
+This folder has examples of RAG applications with data prep kit (DPK).
+
+## Step-0: Getting Started
+
+```bash
+## Clone this repo
+git  clone   https://github.com/sujee/data-prep-kit
```

Comment:
Once this is merged with the main branch, do we want to maintain this fork ? If we do not maintain the fork, the URL will need to change to https://github.ibm.com/IBM/data-prep-kit

Reply:
@touma-I Typo in the URL above: https://github.com/IBM/data-prep-kit

@sujee, I tried running your branch and got an error. I used Option B in examples/notebooks/rag/setup-python-dev-env.md to setup the Python dev environment and then started rag_1A_dpk_process_ray.ipynb in the Jupyter notebook, but something about the DPK imports is not right and I get the error below:
<img width="1100" alt="image" src="https://github.com/user-attachments/assets/f5812fa5-046c-460f-b583-892540e47f40">

Reply:
> Once this is merged with the main branch, do we want to maintain this fork ? If we do not maintain the fork, the URL will need to change to https://github.ibm.com/IBM/data-prep-kit

Good point.  will update to reflect the main repo!

Reply:
@sujee I assume you don't see this error in your environment, so if you want to test a few other things in my environment to find the source of the problem, please ping me and we can have a quick session of sharing my screen and trying things.

Reply:
@sujee Just did a test of going through all Notebooks (all the way to the end of Step 6 in README) and verified that everything worked. 
@touma-I I think this is ready to pull into dev.